Pkgfile

```diff
@@ -1 +1,35 @@
-# syntax = docker.io/autonomy/bldr:2b3b543-frontend
+# syntax = docker.io/autonomy/bldr:v0.1.0-alpha.0-frontend
+
+format: v1alpha2
+
+vars:
+  BOOTSTRAP: /bootstrap
+
+  binutils_version: 2.33.1
```

Comment:
I got these exposed top-level to avoid duplication between multiple packages using same sources, and so that we can't forget to bump all of them